bundles/org.openhab.binding.modbus.helioseasycontrols/src/main/feature/feature.xml

```diff
@@ -2,7 +2,7 @@
 <features name="org.openhab.binding.modbus.helioseasycontrols-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.4.0">
 	<repository>file:${basedirRoot}/bundles/org.openhab.io.transport.modbus/target/feature/feature.xml</repository>
 
-	<feature name="openhab-binding-modbus-helioseasycontrols" description="Modbus.HeliosEasyControls Binding" version="${project.version}">
+	<feature name="openhab-binding-modbus.helioseasycontrols" description="Modbus.HeliosEasyControls Binding" version="${project.version}">
```

Comment:
This change as was intentional as part of the migration to OH3 and should remain.